packages.x86_64

```diff
@@ -122,15 +119,13 @@ xfsprogs
 xl2tpd
 zsh
 
-# virbos
-#maxfetch
```

Comment:
Could you please leave the maxfetch comment in, because that package is from our custom repo that doesn't work yet (https://github.com/Virbos/pkgs)

Reply:
Alright, ill be editing it to add a commented maxfetch